.github/comment.yml

```diff
@@ -35,11 +35,11 @@
     label: CI-FixRequiredOnFailure
     onLabeledComments: >- 
       Hi @${PRAuthor}, Your PR has some issues. Please fix the CI sequentially by following the order of `Avocado, semantic validation, model validation, breaking change, lintDiff`. 
-      <table><tr><th>Task</th><th>How to fix</th><th>Priority</th><th>Support (Microsoft alias)</th></tr>
-      <tr><td>Avocado</td><td>[Fix-Avocado](https://github.com/Azure/avocado)</td><td>High</td><td>ruowan</td></tr>
-      <tr><td>Semantic validation</td><td>[Fix-SemanticValidation-Error](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#semantic-validation)</td><td>High</td><td>raychen, jianyxi</td></tr>
-      <tr><td>Model validation</td><td>[Fix-ModelValidation-Error](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#model-validation)</td><td>High</td><td>raychen,jianyxi</td></tr>
-      <tr><td>LintDiff</td><td>[Fix-LintDiff](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/ci-fix.md#linter-validation)</td><td>high</td><td>jianyxi, ruoxuan</td></tr></table> 
+      <table><tr><th>Task</th><th>How to fix</th><th>Priority</th><th>Support Channel</th></tr>
```

Comment:
Can we have four columns merged to one column since they have same content - team channel link?

Reply:
It's hard to merge it to one. can remove this column. add link to comments?

Reply:
Sure, let's add link to comment.